static/js/components/AudioPlayer.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+import _ from "lodash"
+import React, { useEffect, useRef, useCallback, useState } from "react"
+import { useSelector } from "react-redux"
+import { currentlyPlayingAudioSelector } from "../lib/redux_selectors"
+import Amplitude from "amplitudejs"
```

Comment:
this is a fiddly thing, but we usually put a blank line to separate outside dependency imports from imports from our own modules